### PR TITLE
UG and DG for menu management

### PR DESCRIPTION
**Pull request by yican95**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -887,6 +906,146 @@ Steps 1b1-1b2 are repeated until the number of units entered is less than the nu
...
+** 3b1. App shows an error message that the given percentage is invalid.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: UC305 - Export menu
+
+*MSS*
+
+1.  User requests to export menu
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: Example of missing fullstop.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -887,6 +906,146 @@ Steps 1b1-1b2 are repeated until the number of units entered is less than the nu
...
+** 3b1. App shows an error message that none of the optional fields are specified.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: UC304 - Give an item a discount
+
+*MSS*
+
+1.  User requests to list items
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: Example of missing fullstop.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -606,6 +606,25 @@ more revenue
 |`* *` |profit-driven restaurant owner |know the most profitable days| employ marketing strategies to bring in more
 revenue
 
+|`* * *` |restaurant owner |add a new item to the menu |
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: Missing the "so I can..." column.

**Comment by yican95** [comment]
Added full stops and the missing the "so I can..." column

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -480,6 +480,187 @@ Format: `history [stockup] [consumption]`
...
+****
+* Deletes the item at the specified `INDEX`.
+* The index refers to the index number shown in the displayed item list.
+* The index *must be a positive integer* 1, 2, 3, ...
+* Alternatively, deletes the item with the specified `NAME`.
+****
+
+Examples:
+
+* `list-item` +
```
Comment (rebstan97) [inline docs/UserGuide.adoc]: change all occurrences of `list-item` to `list-items`

**Pull request merged**